tasks/task_patch.c

```diff
@@ -747,15 +755,32 @@ bool patch_content(
             msg_hash_to_str(MSG_SEVERAL_PATCHES_ARE_EXPLICITLY_DEFINED));
       return false;
    }
-
-   if (     !try_ips_patch(allow_ips, name_ips, buf, size)
+   
+   for (i = 1; i < 10; i++)
```

Comment:
Changing the counter start to 2 and removing the later increment in this loop appears to sort the issue fully.